rhtap-cli-release/rhtap-cli-konflux.sh

```diff
@@ -0,0 +1,800 @@
+        printf "\nCreating branch '$BRANCH' in repository '$GITLAB_ORG/$REPOSITORY'.\n"
+        res=$(eval "$cmd")
+        if [ "$(echo $res | jq -r 'has("error")' 2>/dev/null)" == "true" ]; then
+            message="$(echo $res | jq -r .error)"
+            echo "ERROR: Creating '$BRANCH' - $message"
+            exit 1
+        elif [ "$(echo $res | jq -r 'has("message")' 2>/dev/null)" == "true" ]; then
+            message="$(echo $res | jq -r .message)"
+            echo "ERROR: Creating '$BRANCH' - $message"
+            exit 1
+        fi
+        printf "Creation of branch '$BRANCH' in repository '$GITLAB_ORG/$REPOSITORY' - SUCCESSFUL\n\n"
+    else
+        # Dry run print command
+        printf "\nCMD to create branch '$BRANCH' in repository '$GITLAB_ORG/$REPOSITORY'\n"
+        echo -e "CMD>: $cmd"
+    fi
+}
+
+clone_branch() {
+    # Create Clone command for new branch
+    cmd="git clone --quiet -b $BRANCH https://$AUTH_HEADER@gitlab.cee.redhat.com/$GITLAB_ORG/$REPOSITORY.git /tmp/$BRANCH_DIR 2>&1"
+
+        # Clone branch
+        echo "Cloning of '$GITLAB_ORG/$REPOSITORY.git' branch '$BRANCH' into directory '$BRANCH_DIR'"
+        res=$(eval "$cmd")
+
+        # Successful clone then fetch latest else fail
+        if [ $? -eq 0 ] && [ "$(ls -A /tmp/$BRANCH_DIR 2>/dev/null | wc -l)" -ne 0 ]; then
+            echo "Cloning of '$GITLAB_ORG/$REPOSITORY.git' branch '$BRANCH' into directory '$BRANCH_DIR' - SUCCESSFUL"
+            cd /tmp/$BRANCH_DIR
+            git fetch
+        else
+            echo "Error: Cloning of $GITLAB_ORG/$REPOSITORY.git failed - $res"
+            exit 1
+        fi
+}
+
+update_file_content() {
+    if [[ "$FILE_DIR" == "tenants-config/cluster/stone-prd-rh01/tenants/rhtap-shared-team-tenant" ]]; then
+        export STREAM_NAME="rhtap-cli-release-$MOD_VERSION"
+        export STREAM_VER="\"$VERSION\""
+        export STREAM_BRANCH="\"release-$VERSION\""
+        yq -i '.metadata.name = env(STREAM_NAME)' $PATH_TO_FILE
+        yq -i '.spec.template.values[0].value = env(STREAM_VER)' $PATH_TO_FILE
+        yq -i '.spec.template.values[1].value = env(STREAM_BRANCH)' $PATH_TO_FILE
+    elif [[ "$FILE_DIR" == "config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/rhtap-shared-team" ]]; then
+        export RPA_NAME="rhtap-cli-$MOD_VERSION"
+        export RPA_VER="\"$VERSION\""
+        export RPA_VER_LONG="\"$VERSION.0\""
+        export RPA_TS="\"$VERSION.0-{{ timestamp }}\""
+        yq -i '.spec.applications[0] = env(RPA_NAME)' $PATH_TO_FILE
+        yq -i '.spec.data.releaseNotes.product_version = env(RPA_VER)' $PATH_TO_FILE
+        yq -i '.spec.data.mapping.components[0].name = env(RPA_NAME)' $PATH_TO_FILE
+        yq -i '.spec.data.mapping.components[0].tags[4] = env(RPA_VER_LONG)' $PATH_TO_FILE
+        yq -i '.spec.data.mapping.components[0].tags[5] = env(RPA_TS)' $PATH_TO_FILE
+    else
+        echo "ERROR: Failed to update $PATH_TO_FILE - No conversion data for file"
+    fi
```

Comment:
After updating file content, could you print out the file content? If something wrong, we can double check based on the updated file content.